topics/contributing/tutorials/create-new-tutorial-content/tutorial.md

```diff
@@ -688,5 +688,82 @@ A bibliography will automatically be appended to the end of your tutorial (scrol
 > If you have a DOI for a paper, you can easily obtain the bibtex citation using [doi2bib.org](https://www.doi2bib.org/).
 {: .tip}
 
+
+# Icons
+
+We use the following icons:
+
+Icon ID                  | Icon
+---                      | ---
+comment                  | {% icon   comment %}
```

Comment:
We could maybe generate this table automatically with a for loop. It would avoid having to update it if we add, remove, change some icons in the config file. What do you think?

Reply:
it was meant mostly as a temporary table for testing purposes that @hexylena  made for me (using a for loop indeed). ..we could keep a table in the tutorial if handy (though most icons are used just by the template, and not necessarily useful for tutorial creators)

Reply:
ah ok 😄